### DISCUSSION TITLE
Repository Move Timeline and Steps

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [okd-project/okd#1245] [General]
This document outlines the timeline and steps necessary to move the okd.io repo to the new repository.

**Comment by binnes** [comment]
I did a test to verify that transferring a GitHub repo from one account to another works:
- all code and branches get transferred
- issues get transferred and kept assignees
- pull requests got transferred and could be merged after the transfer completed
- discussions got transferred 

Forking a repo only pulls the code and branches.  The issues, pull requests and discussions don't come over to the forked repo.

When transferring the repo the recipient gets an e-mail requesting them to accept the repo - so we need to ensure the new account owner is available and can access e-mail when performing the transfer

**Comment by binnes** [comment]
When moving okd.io repo we will need to synchronise with the Red Hat team that manages DNS to redirect https://okd.io traffic to the new location.  The okd.io site may be down for a few minutes during the transfer and DNS update

**Comment by brianinnes** [comment]
Diane contacting Red Hat people to coordinate transfer of repos and update CNAME record.  Once this is coordinated we will schedule the move
